packages/trpc/server/routers/viewer/admin/setSMSLockState.handler.ts

```diff
@@ -16,6 +16,20 @@ type GetOptions = {
 const setSMSLockState = async ({ input }: GetOptions) => {
   const { userId, username, teamId, teamSlug, lock } = input;
   if (userId) {
+    const userToUpdate = await prisma.user.findUnique({
+      where: {
+        id: userId,
+      },
+    });
+    if (!userToUpdate) {
+      throw new TRPCError({ code: "BAD_REQUEST", message: "User not found" });
+    }
+    if (userToUpdate.smsLockState === SMSLockState.UNLOCKED) {
+      throw new TRPCError({
+        code: "BAD_REQUEST",
+        message: "Re-review for unlocked user or team is not allowed",
+      });
+    }
```

Comment:
Nah, this is not correct. Also, you checked whether the user's smslockstate is unlocked or not, but all users' smslockstate UNLOCKED by default in the database.

Reply:
Done.